index.html

```diff
@@ -63,5 +64,250 @@ <h1>
+   <ul class="top-list">
+       <li>CANOPY</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+       <li>Back End Dev</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+   </ul>
+
+  </div>
+   <p>
+       A daily selection of privately personalized 
+       reads; no accounts or 
+       sign-ups required.
+   </p>
+
+   <ul class="list-bottom">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+   </ul>
+   <div class="action">
+     <button type="button" class="button"> See Project </button>
+   </div>
+   
+</section>
+</section>
+
+<!-- mobile section  CARD 4  -->
+
+<section class="section-nav card">
+
+  <h2 class="header-mobile">
+       <img src="images/section_header2.png" alt="logo">
+  </h2>
+
+<section class="section-mobile">
+
+  <div class="project-title space-list">
+
+     <h2>Multi-Post <br>
+      Stories</h2>
+     
+   <ul class="top-list">
+       <li>CANOPY</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+       <li>Back End Dev</li>
+       <li><img src="images/Counter.png" alt="dot-counter"></li>
+   </ul>
+
+  </div>
+   <p>
+       A daily selection of privately personalized 
+       reads; no accounts or 
+       sign-ups required.
+   </p>
+
+   <ul class="list-bottom">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+   </ul>
+   <div class="action">
+     <button type="button" class="button"> See Project </button>
+   </div>
+   
+</section>
+</section>
+
+<!-- About myself -->
+
+<section class="about-myself">
+<div class="about-container">
+  <h2>
+    About <br>
+    Myself
+  </h2>
+  <p>
+    Hello I’m a software developer! I can help
+     you build a product , feature or website 
+     Look through some of my work and
+      experience! If you like what you see and
+       have a project you need coded, don’t
+        hestiate to contact me.
+  </p>
+  <div class="social-media">
+    
+          <p class="social-text">LET’S CONNECT</p>
+    
+          <ul class="media-icons">
+              <li> <a href="https://twitter.com/PamphileMusonda">
+                <img src="images/twitter.png" alt="twitter"> 
+              </a> </li>
+
+              <li> <a href="https://linkedin.com/in/PamphileMusonda">
+                <img src="images/LinkedinIcon.png" alt="linkedin"> 
+              </a> </li>
+
+              <li> <a href="">
+                <img src="images/Vector(1).png" alt="twitter"> 
+              </a> </li>
+              
+              <li> <a href="">
+                <img src="images/Vector(2).png" alt="twitter"> 
+              </a> </li>
+
+              <li>  <a href="">
+                <img src="images/Vector(3).png" alt="twitter"> 
+              </a> </li>  
+          </ul>    
+  </div>
+  <div class="get-myResume" >
+    <p>Get my resume</p>
+  </div>
+
+  <ul class="language">
+    <li><p>Languages</p></li>
+    <li><img src="images/Union2.png" alt="left arrow"></li>
+  </ul>
```

Comment:
[OPTIONAL] It seems to me you misunderstood this part of the assignment. This content should be wrapped in a single `<ul>` tag, with each category wrapped inside a `<li>` tag.

<img width="680" alt="Screen Shot 2022-05-30 at 14 35 53" src="https://user-images.githubusercontent.com/82426682/171051375-67e87c19-dcf6-41ec-9386-35fb5885c017.png">